README.md

```diff
@@ -24,6 +24,7 @@ A web-based GUI to manage a [Todo.txt](http://todotxt.com/) file.
     - English (`en`)
     - French (`fr`)
     - Portuguese (WIP) (`pt`)
+  - Multi user / multi todo if used with WebDav
```

Comment:
Please remove this line as WebDav support is detailed later in the readme. Also increment the number of storage backends supported above.

Reply:
Better mention the multi auth backends support

Reply:
Addressed in latest commits.